core/store/orm/orm.go

```diff
@@ -1338,7 +1351,7 @@ func (orm *ORM) FindEncryptedOCRKeyBundleByID(id string) (*ocrkey.EncryptedKeyBu
 
 // DeleteEncryptedOCRKeyBundle deletes the provided encrypted OCR key bundle
 func (orm *ORM) DeleteEncryptedOCRKeyBundle(key *ocrkey.EncryptedKeyBundle) (err error) {
-	return orm.DB.Delete(&key).Error
+	return orm.DB.Delete(key).Error
```

Comment:
Uff. How did this ever work?

Reply:
idk but it did :laughing: maybe gorm recursively follow pointers or something?